src/locales/ko/mystery-encounters/berries-abound-dialogue.json

```
@@ -1,26 +1,26 @@
 {
-  "intro": "There's a huge berry bush\nnear that Pokémon!",
-  "title": "Berries Abound",
-  "description": "It looks like there's a strong Pokémon guarding a berry bush. Battling is the straightforward approach, but it looks strong. Perhaps a fast Pokémon could grab some berries without getting caught?",
-  "query": "What will you do?",
-  "berries": "Berries!",
+  "intro": "포켓몬 근처에 열매가\n달린 거대한 나무가 있다!",
+  "title": "나무열매 풍년",
+  "description": "나무열매 덤불을 지키고 있는 강력한 포켓몬이 있는 것 같습니다.\n그냥 싸워볼 수도 있겠지만, 조금 강해보입니다.\n혹시 빠른 포켓몬이라면 잡히지 않고 열매를 가져올 수 있지 않을까요?",
```

```suggestion
  "description": "나무열매 덤불을 지키고 있는 강력한 포켓몬이 있는 것 같습니다.\n그냥 싸워 볼 수도 있겠지만, 조금 강해 보입니다.\n혹시 빠른 포켓몬이 있다면 잡히지 않고도 열매를 가져올 수 있지 않을까요?",
```
의미를 조금 명확하게 고쳐 보았습니다.